lib/AST/RawComment.cpp

```diff
@@ -61,71 +61,53 @@ SingleRawComment::SingleRawComment(CharSourceRange Range,
                                    const SourceManager &SourceMgr)
     : Range(Range), RawText(SourceMgr.extractText(Range)),
```

Comment:
RawText and ColumnIndent are the only two fields on RawComment that are actually always valid. Range is only set when it isn't a deserialized RawComment. Ideally we'd remove Range from here as well, but it's used in a fair few places... This change is a middle ground where just StartLine/EndLine are removed.